framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -233,6 +243,16 @@
     /** Parses stub files and stores annotations from stub files. */
     public final AnnotationFileElementTypes stubTypes;
 
+    /** Parses ajava files and stores annotations from ajava files. */
+    public final AnnotationFileElementTypes ajavaTypes;
+
+    /**
+     * If processing a file, stores any annotations read from an ajava file for that class. Unlike
```

Comment:
Should "processing a file" be "processing an ajava file"?  Or does this mean "if the Checker Framework is type-checking a .java file, stores annotations read from an ajava file for that class, if one exists"?